TravelFunds/2019.md

```diff
@@ -3,20 +3,20 @@
 First Name | Last Name | Event | Reason | Trip Report | Location | Travel Dates | Amount Requested: | Pull Request date | Pull Request link | Date Expense report sent | Amount of Expense Report | Date Sent to Finance | Date approved through Bill.com | Bill.com Amount approved for reimbursement
 -|-|-|-|-|-|-|-|-|-|-|-|-|-|-
 Anna | Henningsen | Diagnostics WG Summit | Attendance | N/A | Munich |6 Mar – 9 Mar 2019 | 252.68 € | 21 Jan 2019 | https://github.com/nodejs/admin/pull/295 ||$288.56|22 Mar 2019||$288.56
-Rich | Trott | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1404.12 | 8 Mar 2019 | https://github.com/nodejs/admin/pull/309 ||$1,404.12|2 Jul 2019||
+Rich | Trott | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1404.12 | 8 Mar 2019 | https://github.com/nodejs/admin/pull/309 ||$1,404.12|2 Jul 2019|12 Jul 2019|$1404.12
 Guy | Bedford | Collaborator Summit | Attendance || Berlin |30 May - 31 May 2019 | $1185 USD | 13 March 2019 |||$1,185.16|20 Jun 2019|23 Jun 2019| $1,185.16
 Anto | Aravinth | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 1300 USD | 21 March 2019 |||$1,027.77|20 Jun 2019|23 Jun 2019 | $1,027.77
 Ruben | Bridgewater | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 650 € | 28 March 2019 | https://github.com/nodejs/admin/pull/322 |||||
-Mohit | Bajoria | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 1651 USD | 28 March 2019 | https://github.com/nodejs/admin/pull/323 ||$1,594|27 Jun 2019||
-Benjamin | Gruenbaum | Collaborator Summit | Attendance | | Berlin | 30 May - 31 May 2019 | 800 USD | 28 March 2019 |||$800|28 Jun 2019||
-Tobias | Nießen | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 500 € | 9 April 2019 | https://github.com/nodejs/admin/pull/333 ||$558.33|27 Jun 2019||
-Dhruv | Jain | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 2000 USD | 5 April 2019 | https://github.com/nodejs/admin/pull/331 ||$1,817.03|26 Jun 2019||
+Mohit | Bajoria | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 1651 USD | 28 March 2019 | https://github.com/nodejs/admin/pull/323 ||$1,594|27 Jun 2019|12 Jul 2019|$1,594
+Benjamin | Gruenbaum | Collaborator Summit | Attendance | | Berlin | 30 May - 31 May 2019 | 800 USD | 28 March 2019 |||$800|28 Jun 2019|$800|12 Jul 2019
```

Comment:
Amount and date are swapped

Reply:
Thanks, this is fixed in the patch I just pushed.